dist/ge-1-1-6/samples/entando-tenant-ingress.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    entando.org/YOUR-APP-NAME-appbuilder-path: /app-builder/
+    entando.org/YOUR-APP-NAME-de-path: /digital-exchange
+    entando.org/YOUR-APP-NAME-pn-3c07adf0-fac54a9f-entando3323-pn-3c07adf0-fa-path: /entando-epc-bootstrap-3c07adf0/app-builder-menu-bff
```

Comment:
@pietrangelo @eugeniosant If the app name is long (see @avdev4j 's comment), you hit the 63 character limit very easily and the ingress cannot be created. What is required here?

Reply:
Plan is to remove the bff references since it's not used in secondary tenants.

Reply:
Removing just the annotation appears to have no side effects